Source/DafnyCore/CoverageReporter/CoverageReportCommand.cs

```diff
@@ -13,6 +13,9 @@ namespace DafnyCore.CoverageReporter;
 
 public class CoverageReportCommand : ICommandSpec {
 
+  public static List<string> CoverageReportsToMerge = new();
```

Comment:
Ah, but now you need statics, which aren't great either.  

Could you allow generically storing arguments in DafnyOptions. I put a draft of this idea in this PR: https://github.com/dafny-lang/dafny/pull/4410/files ?

Reply:
Thank you, this has been super helpful!

Reply:
Woop :) thanks. Closing the PR then.